### ISSUE TITLE
MSON - array|enum - members description

### ISSUE THREAD

**Issue by klokane** [apiaryio/drafter#96]
## input
```
# Example

## GET /
+ Attributes

    + id: 21 (enum[number], optional) - Database ID
        + Members
            + 37 (number) - Testing value
            + 23 - should be number because of `enum[number]`
            
+ Response 201
```

## response (partial)
```
....
                    "value": {
                              "element": "enum",                                                   
                              "content": [                                                         
                                { 
                                  "element": "number",                                             
                                  "content": 21                                                    
                                },                                                                 
                                { 
                                  "element": "number",                                             
                                  "content": 37                                                    
                                },                                                                 
                                { 
                                  "element": "number",                                             
                                  "content": 23                                                    
                                }
```

## bug
- elements with values 23 && 37 should contain `meta.description` for blueprint above

**Comment by pksunkara** [comment]
Hey, do you want to change `enum[number]` to `enum[number, string]` since you have a `string` value in `Members`?

**Comment by klokane** [comment]
Ok, to simplify case will remove line 
```
            + 1  (string) 
```

Will update uriginal example.

**Comment by pksunkara** [comment]
Fixed by #102
